app/src/main/java/edu/uc/kovaciad/slicetracker/dao/IBrandDAO.kt

```diff
@@ -17,8 +15,21 @@ interface IBrandDAO {
 
     @Insert
     fun insertBrand(brand: Brand)
+    /**
+     * When updating a row with a value already set in a column,
+     * replaces the old value with the new one.
+     *
+     * @param brand new value to write
+     */
+    @Update
+    suspend fun update(brand: Brand)
```

Comment:
Caution: introducing the suspend function causes the build to fail.  See the CircleCI output for the error message.